pkg/util/config/config.go

```diff
@@ -123,3 +129,13 @@ func Copy(c *Config) Config {
 
 	return cfg
 }
+
+func validate(cfg *Config) (err error) {
+	if cfg.MinInstances > 0 && cfg.MaxInstances > 0 && cfg.MinInstances > cfg.MaxInstances {
+		err = fmt.Errorf("minimum number of instances %d is set higher than the maximum number %d")
+	}
+	if cfg.Workers == 0 {
+		err = fmt.Errorf("number of works should be higher than 0")
+	}
+	return
+}
```

Comment:
Nothing really wrong with this, but I'm curious: why check all errors but only panic on the last one? (not really important as there are just two, but could this grow in the future?  it would probably be most helpful to log all of them and then panic)

Reply:
Yes, didn't care much about that for now (also, there are many other parameters that could be validated), just picked up a low-hanging fruit in the config to validate, since we had no validation before altogether.